src/msw/renderer.cpp

```diff
@@ -604,6 +604,11 @@ int wxRendererMSW::GetHeaderButtonHeight(wxWindow * WXUNUSED(win))
 
     wxON_BLOCK_EXIT1( ::DestroyWindow, hwndHeader );
 
+    // Must ensure the proper font is set or the wrong value will be returned
+    // At 200% scaling it was returning a height of 28 when it should have been 40
```

Comment:
I think this line of this comment is not really necessary, a call to `WM_SETFONT` is clearly missing here (thanks for noticing it), it's enough to just say

     // Set the font, even if it's the default one, before measuring the window.

i.e. what the first comment line already does, more or less.